integrations/vite/nuxt.test.ts

```diff
@@ -37,42 +36,62 @@ const SETUP = {
   },
 }
 
-test('dev mode', SETUP, async ({ fs, spawn, getFreePort }) => {
-  let port = await getFreePort()
-  await spawn(`pnpm nuxt dev --port ${port}`)
+test('dev mode', SETUP, async ({ fs, spawn, expect }) => {
+  let process = await spawn('pnpm nuxt dev', {
+    env: {
+      TEST: 'false', // VERY IMPORTANT OTHERWISE YOU WON'T GET OUTPUT
```

Comment:
Vitest (I think) sets a `process.env.TEST` environment variable. If this is set to `true`, then `nuxt dev` doesn't show any output. Why? I'm not sure, but it was confusing because it worked while running manually. It also worked with copying the `spawn()` setup in a custom `whatisgoingon.js` file...